src/sentry/seer/autofix.py

```diff
@@ -808,6 +816,11 @@ def trigger_autofix(
 
     group.update(seer_autofix_last_triggered=timezone.now())
 
+    # log billing event for seer autofix
+    quotas.backend.record_seer_run(
```

Comment:
So, if we set it up like this does it mean that we count a seer run for each time you press start on autofix, including:
1. Hit start on a fresh issue without an autofix
2. Hit start over then start on an issue with an autofix
3. Automation that calls this method
but not if say, you do a rethink from the very beginning of an Autofix?

Reply:
Yep that's the intention